ui/app/components/transit-key-actions.hbs

```diff
@@ -109,7 +108,7 @@
       @submitIsRunning={{this.doSubmit.isRunning}}
       data-test-transit-action={{@selectedAction}}
     />
-  {{else if (or (eq @selectedAction "export") (eq @key.supportedActions.firstObject "export"))}}
+  {{else if (or (eq @selectedAction "export") (eq (get @key.supportedActions 0) "export"))}}
```

Comment:
I definitely could be wrong..but I thought the index had to be passed as a string in the template. 
```suggestion
  {{else if (or (eq @selectedAction "export") (eq (get @key.supportedActions "0") "export"))}}
```